src/modules/services_manager_target.rb

```diff
@@ -68,7 +68,6 @@ def initialize
     end
 
     # @return [Hash] Collection of available targets
-    # @example {'rescue' => {:enabled=>false, :loaded=>true, :active=>false, :description=>'Rescue'}}
```

Comment:
I think we can do better here, I'll make the fix

Reply:
You are most welcome to do that. This must have been broken for a longer time.